juriscraper/opinions/united_states/federal_appellate/ca7.py

```diff
@@ -22,6 +22,13 @@ def _process_html(self):
             docket = parts[parts.index("case#") + 1]
             name = item["summary"].split(docket)[1].split("(")[0]
             author = item["summary"].split("{")[1].split("}")[0]
+            date = (
+                item["published"]
+                if item["published"] not in ["", None, "::"]
+                else None
+            )
+            if not date:
+                continue
```

Comment:
there are other fields in the JSON feed specifically 
`published_parsed` that is none in the failed data.  It looks like an empty non existent file so let's just check if this field is none at the start of the loop and just throw a bad data warning or something if we come across it. 
```
            if item.get("published_parsed", None) is None:
                logger.warning("BAD DATA")
                continue
```